CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 - Fixed tensorboard failing to sync when logging batches of images. (@jacobromero in https://github.com/wandb/wandb/pull/8641)
 - Fixed behavior of `mode='x'`/`mode='w'` in `Artifact.new_file()` to conform to Python's built-in file modes (@tonyyli-wandb in https://github.com/wandb/wandb/pull/8553)
 - Do not ignore parameter `distribution` when configuring sweep parameters from SDK. (@temporaer in https://github.com/wandb/wandb/pull/8737)
+- Fix offline logging of tables by defaulting to backend accepting client IDs. To revert to old behavior, use setting `allow_offline_artifacts = False`. (@domphan-wandb in https://github.com/wandb/wandb/pull/8792)
```

Comment:
could you please move this to the unreleased section. 
Also maybe it makes sense to put it under `Changed` since it changes the existing functionality or default behavior?

Reply:
crap didn't even realize it moved to a different section. 👍

Reply:
sounds good! i think i fix like 80% of PRs where people miss this! :/

Reply:
https://github.com/wandb/wandb/pull/8827